tests/integration/data/test-bundle-external-certs.yaml

```diff
@@ -0,0 +1,31 @@
+# Copyright 2025 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+name: integration-test-external-certs
+description: |-
+  Used to deploy or refresh within an integration test model
+series: jammy
+applications:
+  self-signed-certificates:
+    charm: self-signed-certificates
+    constraints: cores=1 mem=4G root-disk=16G
+    num_units: 1
+  k8s:
+    charm: k8s
+    num_units: 1
+    constraints: cores=2 mem=8G root-disk=16G
+    options:
+      bootstrap-certificates: external
+      bootstrap-node-taints: "node-role.kubernetes.io/control-plane=:NoSchedule"
+  k8s-worker:
+    charm: k8s-worker
+    constraints: cores=2 mem=8G root-disk=16G
+    num_units: 1
+    options:
+      bootstrap-certificates: external
```

Comment:
I am curious here... what's the cluster look like if you deploy 2 worker apps, one self-signed, one external.  Does that make sense?

Reply:
Yeah, I see the problem. I can leverage the cluster relation to infer the certificates, thanks for the suggestion.